Coordinator/Coordinator/Coordinators/AuthCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+// AuthCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// AuthCoordinator
+final class AuthCoordinator: BaseCoordinator {
+    // MARK: - Public Properties
```

Comment:
```suggestion
    // MARK: - Public Properties
```
Не визуальные компоненты?